rules/guides_rules.json

```
@@ -1,5 +1,140 @@
+          "correct": "Click **Settings**, then **API Keys**.",
+          "incorrect": "Go to the configuration section and look for the relevant part.",
+          "explanation": "The incorrect version is vague, potentially leading to confusion. The correct version is precise and unambiguous."
+        },
+        {
+          "rule": "RW1.4",
+          "correct": "Open the terminal and run `yarn dev` to start the server.",
+          "incorrect": "The server was started by running the `yarn dev` command in the terminal.",
+          "explanation": "The correct version uses active voice and present tense, which improves clarity and engagement."
+        }
+      ]
+    },
+    "headings": {
+      "rules": [
+        {
+          "id": "RH1.1",
+          "description": "Write the title in gerund form to clearly state the user's goal (e.g., 'Setting up your development environment')."
+        },
+        {
+          "id": "RH1.2",
+          "description": "Include 'Before you begin' as top-level headings when they apply."
+        },
+        {
+          "id": "RH1.3",
+          "description": "Include 'Instructions' as top-level headings."
+        },
+        {
+          "id": "RH1.4",
+          "description": "Format step headings as '### Step N - Descriptive title' and increment N sequentially without skipping numbers.",
+          "validation_heuristic": "Check that each step heading starts with '### Step ' and the numeric sequence is continuous."
+        },
+        {
+          "id": "RH1.5",
+          "description": "If the procedure requires only one step, omit the step heading and list the actions directly under 'Instructions'."
+        },
+        {
+          "id": "RH1.6",
+          "description": "Do not repeat heading names anywhere in the document."
+        }
+      ],
+      "examples": [
+        {
+          "rule": "RH1.1",
+          "correct": "Setting up your development environment",
+          "incorrect": "Setup development environment"
+        },
+        {
+          "rule": "RH1.2",
+          "correct": "## Before you begin",
+          "incorrect": "## Pre requisites"
+        },
+        {
+          "rule": "RH1.3",
+          "correct": "## Instructions",
+          "incorrect": "## Instructions and Steps"
+        },
+        {
+          "rule": "RH1.4",
+          "correct": [
+            "### Step 1 - Install dependencies",
+            "### Step 2 - Configure environment"
+          ],
+          "incorrect": [
+            "### Step 1 - Install dependencies",
+            "### Step 3 - Configure environment"
+          ],
+          "explanation": "The incorrect example skips Step 2, breaking the sequential order."
+        },
+        {
+          "rule": "RH1.5",
+          "correct": "## Instructions 1. Run the installation script. 2. Confirm the output.",
+          "incorrect": "## Instructions ### Step 1 - Run the installation script 1. Run the installation script.",
```

Adicionar quebra de linha nos exemplos
```suggestion
          "correct": "## Instructions\n\n1. Run the installation script.\n2. Confirm the output.",
          "incorrect": "## Instructions\n\n### Step 1 - Run the installation script\n\n1. Run the installation script.",
```